### DISCUSSION TITLE
Default values for SecurityScheme

### DISCUSSION THREAD

**Discussion by open-schnick** [juhaku/utoipa#783] [Q&A] [answered]
Hey is there a way i can provide default values for the used SecuritySchema?
So that i only have to press a button in the authenticate-popup and I am authenticated?

**Comment by juhaku** [comment]
As what I understand from the specification there are no such fields available. https://spec.openapis.org/oas/latest.html#security-scheme-object

**Reply by open-schnick** [reply]
Hey, thanks for your answer. I am not that deep into that topic, but Spring Boot somehow does it so that the swagger ui already has the credentials prefilled.

**Reply by juhaku** [reply, marked as answer]
Okay, Probably there is a way to do this that I am not aware of. I don't know the SpringDoc's implementation details thus it is hard to say that how the have solved this. Maybe it is some extension field. This is probably something that could be investigated in future however now I don't have bandwidth to investigate this matter. Though as always PR:s are welcome in that matter. :slightly_smiling_face:
